ml-agents/mlagents/trainers/torch/utils.py

```diff
@@ -56,8 +56,9 @@ def create_encoders(
         h_size: int,
```

Comment:
This is only for vector observations right? I wonder if this create_encoders should create a hidden layer for vector_observations. I think most of the time, we simply concatenate the normalized vector_observations (without an additional layer) with the visual observations encodings.

Reply:
I'm pretty sure in TF we add some hidden layers to the vector_observations before concatenating with visual observations - or do you mean the visual_observations? I did find a bug where we're not adding any hidden layers to the visual obs in Torch.